int16 auto_manual
int16 emergency_stop
int16 gear
float32 speed
float32 steer
int16 brake
float32 encoder

